src/Batteries.purs

```diff
@@ -357,11 +337,7 @@ import Control.Monad.Aff.Class
   ( class MonadAff
   , liftAff
   )
--- import Control.Monad.Aff.Console -- NOTE: Math, Control.Monad.Eff.Console
-import Control.Monad.Aff.Par
-  ( Par(Par)
-  , runPar
-  )
+-- import Control.Monad.Aff.Console -- NOTE: Control.Monad.Eff.Console
```

Comment:
Note: it _should_ conflict with `eff` (whose functions were renamed in `0.8`) but _does_ conflict with `math`.